NMS-14212: Event docs cleanup (#5002)
OpenNMS/opennms: docs/modules/operation/pages/alarms/configuring-alarms.adoc

  </restriction>
</simpleType>
----

> NOTE: See also: <<events/introduction.adoc#events, anatomy of an event>>

Author: ```suggestion
NOTE: See also: <<<events/event-definition.adoc#ga-events-anatomy-of-an-event, anatomy of an event>>
```

</simpleType>
----

> *NOTE*

Author: ```suggestion
```

> See also: <<events/introduction.adoc#events, anatomy of an event>>

Reviewer: ```suggestion
NOTE: See also: <<events/introduction.adoc#events, anatomy of an event>>
```
